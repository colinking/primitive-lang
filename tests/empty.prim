// SYNTAX TEST "source.prim" "empty file"
