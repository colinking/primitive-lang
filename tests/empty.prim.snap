>// SYNTAX TEST "source.prim" "empty file"
#^^ source.prim comment.line.double-slash.prim punctuation.definition.comment.prim
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.prim comment.line.double-slash.prim
>